__tests__/account.test.ts

```diff
@@ -113,7 +112,7 @@ describe('deploy and test Wallet', () => {
     // this is tested indirectly true declareAndDeploy while declaring
   });
 
-  describeIfDevnetSequencer('Test on Devnet Sequencer', () => {
+  describeIfRpcDevnet('Test on RPC Devnet', () => {
```

Comment:
```suggestion
  describeIfDevnet('Test on RPC Devnet', () => {
```

and to change this in all places? ping @penovicp 🙏

Reply:
Sure, I see no issue with renaming it.